manim/mobject/types/image_mobject.py

```diff
@@ -22,6 +22,13 @@
 class AbstractImageMobject(Mobject):
     """
     Automatically filters out black pixels
+
+    Parameters
+    ----------
+    scale_to_resolution : :class:`int`
+        At this resolution the image is placed pixel by pixel onto the screen, so it will look the sharpest and best.
+        This is a custom parameter of ImageMobject so that rendering a scene with the `--quality low` flag for faster rendering and testing won't effect the position of the image on the screen.
+        Calculated by `height = image_height / scale_to_resolution * config["frame_height"]`
```

Comment:
This description contains a lot of information that is not actually required. I'd suggest to keep the descriptions as concise as possible. If someone wants to know implementation details, they'll have to read the source code anyways.

```suggestion
    Parameters
    ----------
    scale_to_resolution : :class:`int`
        The resolution at which the image is placed on the screen.
```